queue/src/fiskaltrust.Middleware.Localization.QueueDEFAULT/QueueDEFAULTConfiguration.cs

```diff
@@ -6,6 +6,8 @@ namespace fiskaltrust.Middleware.Localization.QueueDEFAULT
     public class QueueDEFAULTConfiguration
     {
         public bool Sandbox { get; set; } = true;
+        public bool FlagOptionalSignatures { get; set; }
```

Comment:
This parameter we don't need in the Default queue. There are no optional signatures here 😅